src/applications/login/containers/MockAuth.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import LoginInfo from 'platform/user/authentication/components/LoginInfo';
+import MockAuthButton from '../components/MockAuthButton';
+
+export default function MockAuth() {
+  return (
+    <section className="login">
+      <div className="container">
+        <div className="container">
+          <div className="row">
+            <div className="columns small-12">
+              <h1
+                id="signin-signup-modal-title"
+                className="vads-u-margin-top--2 vads-u-color--gray-dark medium-screen:vads-u-margin-top--1 medium-screen:vads-u-margin-bottom--2"
+              >
+                Mocked Auth
+              </h1>
+            </div>
+          </div>
+          <div className="row">
+            <div className="columns small-12" id="sign-in-wrapper">
+              <MockAuthButton />
+            </div>
+          </div>
+          <LoginInfo />
```

Comment:
You can remove the <LoginInfo /> component.  Replace with:

Paragraph: 
Mocked authentication is only available in the development and localhost environments. Read more about [mock authentication](#) and how to use it on GitHub.

Once we have mock authentication documentation we can come back and update the URL.